lupa/lua.pxd

```diff
@@ -455,3 +455,16 @@ cdef extern from *:
     """
     int read_lua_version(lua_State *L)
     int lua_isinteger(lua_State *L, int idx)
+
+
+cdef extern from "stdint.h":
```

Comment:
Sadly, `stdint.h` is missing from certain older MSVC versions, apparently before VS 2010. And it's a C99 thing. Let's try not to rely on it. Why is `PTRDIFF` the right fallback here?

Reply:
`lua_Integer` is `ptrdiff_t` in versions of Lua prior to 5.3